### ISSUE TITLE
作者训练好imagenet 模型

### ISSUE THREAD

**Issue by HHEjie123** [jianghaojun/CondenseNetV2#3]
您好，请问我能用作者训练好imagenet 模型进行迁移学习吗，我并没有发现该模型，因为我简单的把该结构拿出来进行图像检索，具体操作为去掉最后的分类层，添加l连接层(1024,128)发现结果并不是很好，大概是我做错了，而且我发现收敛的很慢，运行同样的epoch和损失函数，该网络65个epoch时只有50%,而imagenet预训练下的alexnet都有70%，实验设备有限，我希望能快速得到结果，希望作者能给合适imagenet模型，邮箱是129963018@qq.com,万分感谢。这是我修改的地方

class CondenseNetV2(nn.Module):
    def __init__(self, args, bit):

        super(CondenseNetV2, self).__init__()

        self.stages = args.stages
        self.growth = args.growth
        assert len(self.stages) == len(self.growth)
        self.args = args
        self.progress = 0.0
        if args.dataset in ['cifar10', 'cifar100']:
            self.init_stride = 1
            self.pool_size = 8
        else:
            self.init_stride = 2
            self.pool_size = 7

        self.features = nn.Sequential()
        ### Initial nChannels should be 3
        self.num_features = 2 * self.growth[0]
        ### Dense-block 1 (224x224)
        self.features.add_module('init_conv', nn.Conv2d(3, self.num_features,
                                                        kernel_size=3,
                                                        stride=self.init_stride,
                                                        padding=1,
                                                        bias=False))
        for i in range(len(self.stages)):
            activation = 'HS' if i >= args.HS_start_block else 'ReLU'
            use_se = True if i >= args.SE_start_block else False
            ### Dense-block i
            self.add_block(i, activation, use_se)
#*****************************************************************只修改了这里
        #self.fc = nn.Linear(self.num_features, args.fc_channel)
        #self.fc_act = HS()
        self.hash = nn.Linear(self.num_features, bit)
        self.fc_act = HS()
        ### Classifier layer
        #self.classifier = nn.Linear(args.fc_channel, args.num_classes)
        self._initialize()
#*****************************************************************************************

  def cdnv2_b(self,args, bit):#这里没改，就加个bit
      args.stages = '2-4-6-8-6'
      args.growth = '6-12-24-48-96'
      print('Stages: {}, Growth: {}'.format(args.stages, args.growth))
      args.stages = list(map(int, args.stages.split('-')))
      args.growth = list(map(int, args.growth.split('-')))
      args.condense_factor = 6
      args.trans_factor = 6
      args.group_1x1 = 6
      args.group_3x3 = 6
      args.group_trans = 6
      args.bottleneck = 4
      args.last_se_reduction = 16
      args.HS_start_block = 2
      args.SE_start_block = 3
      args.fc_channel = 1024
      return CondenseNetV2(args, bit)

#************************************************args参数只传了这些，其他都没改
  import os
  import argparse
  import warnings
  warnings.filterwarnings("ignore")
  parser = argparse.ArgumentParser(description='PyTorch Condensed Convolutional Networks')
  args, unknown = parser.parse_known_args()
  args.dataset = 'cifar100'
  args.num_classes = 100
cdnv2_b(args, 64):

**Comment by HHEjie123** [comment]
cifar数据集

**Comment by jianghaojun** [comment]
同学您好，预训练的CondenseNetV2-A/B/C的模型已经提供了，提供的是将SFR模块转化为分组卷积的模型，请使用[Converted_CondenseNetV2](https://github.com/jianghaojun/CondenseNetV2/blob/main/model/converted_condensenetv2.py)来加载提供的预训练参数，应该会有更好的效果！
